.github/workflows/visual-tests-schedule.yml

```diff
@@ -0,0 +1,83 @@
+name: Playwright Visual Tests (Scheduled)
```

Comment:
Should we run tests only in Chrome Desktop on Linux? I don't think we need to run tests in multiple platforms. If it works on Chrome Linux (or any other easy to run in CI platform), then it will work everywhere.

Reply:
we could run them on a single browser, I opted to minimize the changes needed in the playwright config, and these tests seem quite stable, so I wouldn't mind running them on the 3 defined browsers